src/test/java/keycontacts/model/lesson/RegularLessonTest.java

```diff
@@ -78,4 +78,14 @@ public void hashCodeMethod() {
         assertEquals(Objects.hash(regularLesson.getLessonDay(), regularLesson.getStartTime(),
                         regularLesson.getEndTime()), regularLesson.hashCode());
     }
+
+    @Test
+    public void isOnDayAndTimeMethod() {
+        RegularLesson regularLesson = ALICE.getRegularLesson();
+        Day lessonDay = new Day("Monday");
+        Time lessonTime = new Time("12:00");
```

Comment:
Perhaps get day and time from `regularLesson.getLessonDay()` and `regularLesson.getStartTime()`? (Imagine if someone changes ALICE and has to wonder why a test case is suddenly failing)